packages/vue-instantsearch/src/components/Hits.js

```diff
@@ -0,0 +1,57 @@
+import { createHits } from 'instantsearch-jsx';
+import { connectHitsWithInsights } from 'instantsearch.js/es/connectors';
+
+import { createSuitMixin } from '../mixins/suit';
+import { createWidgetMixin } from '../mixins/widget';
+import { renderCompat } from '../util/vue-compat';
+
+const augmentH = (baseH) => (tag, propsWithClassName, children) => {
+  const { className, ...props } = propsWithClassName;
+  return baseH(tag, Object.assign(props, { class: className }), [children]);
+};
```

Comment:
This is to map `className` to `class` and children has to be an array for Vue's `h` function

Reply:
vue 2 and 3 are quite different, we have renderCompat, but I wonder if we should just have separate files for vue 2 and vue 3 once we simplify the rendering?